stan/math/prim/prob/scaled_inv_chi_square_lcdf.hpp

```diff
@@ -40,13 +35,18 @@ return_type_t<T_y, T_dof, T_scale> scaled_inv_chi_square_lcdf(
                          "Degrees of freedom parameter", nu, "Scale parameter",
                          s);
 
+  if (size_zero(y, nu, s)) {
+    return 0;
+  }
+
+  T_partials_return P(0.0);
+  operands_and_partials<T_y, T_dof, T_scale> ops_partials(y, nu, s);
+
```

Comment:
[optional]
I don't understand all the vertical space placement.  I'd prefer to just get rid of almost all the blank vertical lines in this code.  At the very least, we should get rid of spaces between contiguous blocks of variable declarations.